app/controllers/distributions_controller.rb

```diff
@@ -34,4 +34,16 @@ def testing
     flash[:notice] = _('Help test the next version of openSUSE Leap!')
     render action: "leap-#{@testing_version}", layout: 'download'
   end
+
+  # GET /distributions/legacy
+  def legacy
+    @hide_search_box = true
```

Comment:
what is this variable for? :thinking:

Reply:
When that variable is true, the search box won't be displayed on the page.

We only have the search box on `/` (`/distributions`) because it is the homepage and users should be able to start searching right away. Eventually we want to make the new appstore the homepage at which point we can probably remove the search box from any distribution page.